cmd/gh-action-integration-generator/main.go

```diff
@@ -56,9 +56,12 @@ jobs:
             config-example.yaml
 
       - name: Run {{.Name}}
+        uses: Wandalen/wretry.action@master
         if: steps.changed-files.outputs.any_changed == 'true'
         run: |
-            nix develop --command -- docker run \
+        with:
+          attempt_limit: 5
+          command: |
```

Comment:
I think this removes a bit too much?
```suggestion
        with:
          attempt_limit: 5
          command: |
            nix develop --command -- docker run \
```